velox/exec/SharedArbitrator.cpp

```diff
@@ -575,12 +576,16 @@ SharedArbitrator::ScopedArbitration::ScopedArbitration(
 
 SharedArbitrator::ScopedArbitration::~ScopedArbitration() {
   requestor_->leaveArbitration();
-  const auto arbitrationTime =
+  const auto arbitrationTimeUs =
       std::chrono::duration_cast<std::chrono::microseconds>(
-          std::chrono::steady_clock::now() - startTime_);
+          std::chrono::steady_clock::now() - startTime_)
+          .count();
   RECORD_HISTOGRAM_METRIC_VALUE(
-      kMetricArbitratorArbitrationTimeMs, arbitrationTime.count() / 1'000);
-  arbitrator_->arbitrationTimeUs_ += arbitrationTime.count();
+      kMetricArbitratorArbitrationTimeMs, arbitrationTimeUs / 1'000);
+  addThreadLocalRuntimeStat(
+      "memoryArbitrationTime",
```

Comment:
Other similar metrics use xxxNanos or xxxWallNanos naming. Perhaps, rename to memoryArbitrationNanos

Reply:
Updated. Thanks!